compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -284,9 +284,21 @@ xFlagsSet options =
 wOptsSet :: [ WarningFlag ]
 wOptsSet =
   [ Opt_WarnUnusedImports
+-- Can enable when we are on GHC >= 8.10 (we should, after all we
```

Comment:
Not sure we want to. It might cause a fair amount of noise for some projects that haven’t payed attention to this so far and it’s purely a stylistic thing not a bug.

Reply:
Well, of course, I am biased 🤣

Reply:
I would enable it on _my_ projects but that’s different from throwing potentially thousands of warnings at our users :slightly_smiling_face:

Reply:
Oh wow! We almost never agree on these sorts of things! 🤣